.vscode/cspell.json

```diff
@@ -32,6 +32,7 @@
     "azsdk",
     "azurecli",
     "bugbug",
+    "checkpointstore",
```

Comment:
Nit: This is in your `sdk/eventhubs/.dict.txt` file as well. Necessary in both places? If not a problem outside that directory, the `.dict.txt` file would be best for narrowly-scoped words.

Reply:
The problem is the root cargo.toml file :(. So it needs to be in the root. I wish that wasn't the case, but...